src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/ElementsRequestExecutor.java

```diff
@@ -192,10 +192,6 @@ public static void extract(RequestResource requestResource, ElementsGeometry ele
     if (processingData.isContainingSimpleFeatureTypes()) {
       mapAgg = inputProcessor.filterOnSimpleFeatures(mapAgg);
     }
-    Optional<FilterExpression> filter = processingData.getFilterExpression();
```

Comment:
This removal caused the import in line 12 to be obsolete, so it would have to be removed.

Reply:
The obsolete import in this class is still in there.

Reply:
oops, overlooked that one. sorry.